megamek/src/megamek/client/bot/princess/PathRanker.java

```diff
@@ -536,11 +535,8 @@ private boolean willBuildingCollapse(MovePath path, Game game) {
     }
 
     public static @Nullable Coords calcAllyCenter(int myId, @Nullable List<Entity> friends, Game game) {
-        if ((friends == null) || friends.isEmpty()) {
+        if ((friends == null) || friends.size() <= 1) {
```

Comment:
units alone shuold not receive back a "allyCenter", if they have, they will "fight" against the location they currently are, so they won't move around.

Reply:
Does "friends" include the currently moving unit?

Reply:
it does, for absolutelly no good reason other than "we didnt took our time to filter out"